tests/integrated/test-thumbnail.R

```diff
@@ -0,0 +1,90 @@
+# The tests in this file were not written explicitly for the *_thumbnail()
+# functions; they are taken from `tests/integrated/test-deploy.R` where they
+# test the *_image.R functions.
+
+test_that("set_thumbnail works with local images", {
+  scoped_experimental_silence()
+  img_path <- rprojroot::find_package_root_file("tests/testthat/examples/logo.png")
+
+  res <- set_thumbnail(cont1_content, img_path)
```

Comment:
At least some of the integration test issues are because `cont1_content` isn't defined above here (or as far as I can tell anywhere in this file?).

Reply:
Yeah, I noticed. When I originally made this change, I made some incorrect assumptions about the structure of the integration tests.

Reply:
Yeah — I didn't copy the preamble from `integrated/test-deploy.R`